controllers/main.js

```diff
@@ -19,17 +19,21 @@ module.exports.renderPlaySettings = (req, res) => {
 };
 
 module.exports.renderPlay = async (req, res) => {
+  const { id, pageNum } = req.params;
   const image = await getPixelatedImage(req.gameData.background_image);
-  const playSessionData = await fetchPlaySessionData(req.user.username);
-  const pageNum = playSessionData.sessionData.length + 1;
+  console.log(id);
+  const playSessionData = await PlaySession.findOne({ sessionId: id });
+  console.log(playSessionData);
+  const filledPages = playSessionData.sessionData?.length + 1 || 1;
```

Comment:
I'm not entirely sure what does this naming refer to: `filledPages`. Could you explain?

Reply:
pages that are already filled with data (correct game name, game image link)